examples/l3-tcp-syn-flood.lua

```diff
@@ -3,6 +3,7 @@ local memory	= require "memory"
 local device	= require "device"
 local stats		= require "stats"
 
+require "proto.tcp"
```

Comment:
it should not be necessary to explicitly include a protocol if no module-specific stuff is used

i.e. tcp = require "proto.tcp" is okay if tcp.foo needs to be accessed somewhere, otherwise it seems unnecessary